packages/kbn-sandbox/index.test.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+const { fork } = require('child_process');
+
+// we have to spawn a process to actually run the test, or else we end up locking
+// down the test runner process and not letting us spawn anything in sunsequent tests
+test('sandboxes', async (done) => {
+  const cp = fork(path.resolve(__dirname, 'test_process.js'));
```

Comment:
We remove certain test-related files globally from builds, like `*.test.js` and things under `__tests__` directories. Since this test_process.js file doesn't follow any particular convention that we track, it's going to erroneously make its way into builds.

Reply:
Good catch, I'll rename it so it's caught by these.